mat/lu.go

```diff
@@ -250,18 +250,27 @@ func (lu *LU) RankOne(orig *LU, alpha float64, x, y Vector) {
 }
 
 // LTo extracts the lower triangular matrix from an LU factorization.
-// If dst is nil, a new matrix is allocated. The resulting L matrix is returned.
-// LTo will panic if the receiver does not contain a factorization.
+//
+// If dst is empty, LTo will resize dst to be a lower-triangular c×c matrix.
+// If dst is non-empty, LTo will panic if dst is not c×c or if dst is not
+// Lower. LTo will also panic if the receiver does not contain a successful
```

Comment:
"When dst is non-empty, LTo will panic if dst is not c×c or not Lower."

Reply:
Done.